### PR TITLE
Point to pilot-hubs repo for list of hubs

### PR DESCRIPTION
**Pull request by yuvipanda**
- U Toronto hub is listed in https://github.com/2i2c-org/pilot-hubs/pull/413
- The pangeo-hubs repo is defunct, hasn't been actively used in a
  while and will be eliminated soon

### CODE REVIEW

**Comment by yuvipanda** [comment]
Where do we track deliverables for the UToronto hub? Right now, nowhere! 

IMO, we should consider moving that hub into our pilot hubs repo as well. It was the first hub we setup, so it got its own repo. Now maybe we should move it back here. Will make life simpler for us.

File: index.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ Contains information about the 2i2c team and our projects, and some useful resou
 :maxdepth: 1
 :caption: Team Reference
 reference/projects
-reference/hubs
```
Comment (choldgraf) [inline index.md:59]: Imagined that we would leave this page, but just point it to the pilot hubs repo. That way the source of truth is still the team compass, it just delegates to pilot hubs. Maybe we can just add a section to the "team coordination" page?

Comment (yuvipanda) [reply index.md:59]: > Maybe we can just add a section to the "team coordination" page?

Can you expand on that a little?

Comment (choldgraf) [reply index.md:59]: I mean that if we think it is a bit much to have a dedicated page that simply links to the hubs table in this repo, we could turn that page into a short section here https://team-compass.2i2c.org/en/latest/practices/coordination.html

Comment (yuvipanda) [reply index.md:59]: @choldgraf that makes sense - let's do that!

**Comment by choldgraf** [comment]
@yuvipanda is this ready to go?

**Comment by damianavila** [comment]
I am not @yuvipanda 😉  but if we are removing the Hubs page here, we need to make an explicit link/reference to the table living in the pilots-hubs docs. @choldgraf you proposed to put the reference on the coordination page... I think it actually belongs to the https://team-compass.2i2c.org/en/latest/managed-hubs/about.html page, IMHO.

**Comment by choldgraf** [comment]
Good point @damianavila - I've added a reference to that page in our `about` page that you linked above. Also resolved the conflict on `index` so I think we're ready to go unless anybody objects

**Comment by damianavila** [comment]
@choldgraf, I do not actually see the reference you said you added when I look at the docs generated from this PR: https://2i2c-team-compass--103.org.readthedocs.build/en/103/managed-hubs/about.html.
Btw, the last commit is also a little bit weird... are you sure you pushed the change you were intended to push?

**Comment by choldgraf** [comment]
Ermmm yep you're right, will look into it when I'm back at phone

**Comment by choldgraf** [comment]
OK updated the PR! It didn't push properly when I tried earlier, thanks for catching it @damianavila :-)

**Review by damianavila — APPROVED** [review]
LGTM now.

**Comment by choldgraf** [comment]
🚀

**Pull request merged**